src/output_manager/mod.rs

```diff
@@ -25,24 +23,19 @@ impl OutputManager {
     pub fn new() -> Self {
         Self {
             outputs: Default::default(),
-            // space: desktop::Space::new(slog_scope::logger()),
         }
     }
 
     pub fn outputs(&self) -> Ref<Vec<Output>> {
         self.outputs.borrow()
     }
 
-    pub fn add(&mut self, space: &mut Workspace, output: &Output) {
-        let loc = (
-            space
-                .outputs()
-                .fold(0, |acc, o| acc + space.output_geometry(o).unwrap().size.w),
-            0,
-        );
-
-        output.change_current_state(None, None, None, Some(loc.into()));
-        space.map_output(output, 1.0, loc);
+    pub fn add(&mut self, output: &Output) {
+        output.change_current_state(None, None, None, Some((0, 0).into()));
```

Comment:
I'm not a huge fan of advertising false location to the clients here. If we can't get real location, let's just not advertise anything.